extension-manifest-v3/src/store/daily-stats.js

```diff
@@ -0,0 +1,103 @@
+import { store } from 'hybrids';
+import * as IDB from 'idb';
+
+import { registerDatabase } from '/utils/indexeddb.js';
+
+// Synchronously register name of the database
+// so if a user don't open any page, it is still possible
+// to delete the database from settings page
+const DB_NAME = registerDatabase('stats');
```

Comment:
can we use same db name and schema as on MV2 so we wont have to migrate the data?

Reply:
To avoid introducing dexie, we could either drop all old stats in the MV2 -> MV3 move. Or do a one-time migration (in the worst case using the native indexedb API, even it is ugly).

Reply:
Dixie or idb does not limit operations on the db instance - they are "wrapper", which makes handling it easier. The idb is much smaller and less abstract, but you can do everything with it like you would with a plain connection.

More here: https://github.com/ghostery/ghostery-extension/pull/1089#issuecomment-1490139236